cmd/manager/main.go

```diff
@@ -69,6 +76,22 @@ func main() {
 		os.Exit(0)
 	}
 
+	// set TLS to serve metrics over a secure channel if cert is provided
+	// cert is provided by default by the marketplace-trusted-ca volume mounted as part of the marketplace-operator deployment
+	var useTLS bool
+	if *tlsCertPath != "" && *tlsKeyPath == "" || *tlsCertPath == "" && *tlsKeyPath != "" {
```

Comment:
This is a very confusing line of code. Not only did I have to double check Golang's operator precedence, but I misread the alternating comparison operators while trying to understand this. Normally, I would shrug and move on, but for something related to TLS, I'm going to ask that this be rewritten in the simpler form:

```go
if (*tlsCertPath == "") != (*tlsKeyPath == "") {
```